public/js/components/Scopes.js

```diff
@@ -3,30 +3,38 @@ const { bindActionCreators } = require("redux");
 const { connect } = require("react-redux");
 const ImPropTypes = require("react-immutable-proptypes");
 const actions = require("../actions");
-const { getPause, getLoadedObjects } = require("../selectors");
+const { getSelectedFrame, getLoadedObjects, getPause } = require("../selectors");
 const ObjectInspector = React.createFactory(require("./ObjectInspector"));
 const { DOM: dom, PropTypes } = React;
+const toPairs = require("lodash/toPairs");
 
 require("./Scopes.css");
 
 function info(text) {
   return dom.div({ className: "pane-info" }, text);
 }
 
+function excludeVariable(binding) {
+  if (!binding[1]) {
+    return;
+  }
+  const value = binding[1].value;
+  return value.missingArguments || value.optimizedOut;
+}
+
 // Create the tree nodes representing all the variables and arguments
 // for the bindings from a scope.
 function getBindingVariables(bindings, parentName) {
-  return bindings
-    .get("arguments").map(arg => arg.entrySeq().get(0))
-    .concat(bindings.get("variables").entrySeq())
-    .filter(binding => (!binding[1].hasIn(["value", "missingArguments"]) &&
-                        !binding[1].hasIn(["value", "optimizedOut"])))
+  const args = [].concat(...bindings.arguments.map(toPairs));
```

Comment:
Shouldn't need that; the way the data is shaped you can just do `.map(x => toPairs(x)[0]` like before. It's stupid, but it's because the object only ever going to have one key and one value.